src/components/ToggleTodo/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react'
+import CSSModules from 'react-css-modules';
+
+import styles from './styles.css';
+
+const ToggleTodo = ({onClick, actionId, phoneNumber, todo}) => {
+  return (
+    <input type="checkbox"
+           styleName='toggle-todo'
+           onChange={function() {onClick(actionId, phoneNumber)}}
+           checked={todo && todo.completed} />
```

Comment:
Funky, it looks like if `todo` is `undefined`, then the expression `todo && todo.completed` evaluates to `undefined` as well. (And when `checked` evaluates to `undefined`, react thinks the checkbox is uncontrolled.)

One gross way to solve this would be to change the expression to: `todo && todo.completed || false`.

Probably better to extract a variable or even a function at that point?:

```js
<input type="checkbox" checked={shouldCheckToggle(todo)} />

...

function shouldCheckToggle(todo: ?{completed: boolean}): boolean {
  if (todo == null) return false;
  return todo.completed;
}
```